arroyo/processing/strategies/batching.py

```diff
@@ -233,32 +230,3 @@ def __flush(self) -> None:
         logger.debug("Offset commit took %dms", commit_duration)
 
         self.__batch = None
-
-
-class BatchProcessingStrategyFactory(ProcessingStrategyFactory[TPayload]):
-    """
-    Do not use for new consumers.
-    This is deprecated and will be removed in a future version.
-    """
-
-    def __init__(
-        self,
-        worker: AbstractBatchWorker[TPayload, TResult],
-        max_batch_size: int,
-        max_batch_time: int,
-    ) -> None:
-        self.__worker = worker
-        self.__max_batch_size = max_batch_size
-        self.__max_batch_time = max_batch_time
-
-    def create_with_partitions(
-        self,
-        commit: Callable[[Mapping[Partition, Position]], None],
-        partitions: Mapping[Partition, int],
-    ) -> ProcessingStrategy[TPayload]:
-        return BatchProcessingStrategy(
```

Comment:
Are we keeping the `BatchProcessingStrategy` ?

Reply:
Long term no and it is already marked deprecated. Since we are using it currently, i will leave that one around for a bit longer